src/Renderer/LayeredMaterial.js

```diff
@@ -76,7 +90,7 @@ var moveElementsArraySafe = function moveElementsArraySafe(array,index, howMany,
 };
 /* eslint-enable */
 
-const LayeredMaterial = function LayeredMaterial(id) {
+const LayeredMaterial = function LayeredMaterial(material, coord) {
```

Comment:
As some projections are not linear (like PM), it is simpler to do the calculations in the same system and use WMTS coordinate.